clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -444,14 +444,7 @@ public static String stackTrace(Throwable e) {
      * @return The unstarted thread
      */
     public static Thread newThread(String name, Runnable runnable, boolean daemon) {
```

Comment:
In KafkaThread class, Shall I add one overriding constructor `public KafkaThread(final String name)` with daemon as `false` internally ? (In most of the places, the non-daemon thread is being created.)